package.json

```diff
@@ -29,7 +29,7 @@
     "eslint": "~6.1.0",
     "eslint-config-prettier": "^6.1.0",
     "eslint-plugin-babel": "^5.1.0",
-    "eslint-plugin-flowtype": "^2.35.0",
+    "eslint-plugin-flowtype": "^4.2.0",
```

Comment:
do we still need this?

Reply:
peer dep from https://github.com/facebook/fbjs/blob/cf8600398c47dc0b15a2ff036d4cdbcbf1a69191/packages/eslint-config-fbjs/package.json#L25